terraform/modules/azure/managed_identities/main.tf

```diff
@@ -17,4 +17,10 @@ resource "azurerm_user_assigned_identity" "keda_identity_2" {
   name                = "${var.unique_project_name}-e2e-test-identity-2"
   location            = data.azurerm_resource_group.rg.location
   resource_group_name = data.azurerm_resource_group.rg.name
+}
+
+resource "azurerm_user_assigned_identity" "postgres_identity" {
+  name                = "${var.unique_project_name}-e2e-test-postgres"
+  location            = data.azurerm_resource_group.rg.location
+  resource_group_name = data.azurerm_resource_group.rg.name
 }
```

Comment:
To make the things easier, we are using `keda_identity_1` for almost all the things (`keda_identity_2` is only used for a few tests where we want to check custom assignments)

Reply:
That is good to know, thank you!